Art/Source/3D/Sequences/procedures/bp_cuff_apply.py

```
@@ -0,0 +1,249 @@
+    Animate the BP cuff application.
+
+    Args:
+        context: dict with 'hand', 'patient_arm', 'camera', 'bp_cuff', 'bp_bulb'
+        start_time: Start time in seconds
+
+    Returns:
+        float: End time in seconds
+    """
+    hand = context['hand']
+    cuff = context['bp_cuff']
+    bulb = context.get('bp_bulb')
```

Ruff (0.14.7)

Local variable `bulb` is assigned to but never used

Remove assignment to unused variable `bulb`

(F841)